projects/skipper/project.yaml

```diff
@@ -2,6 +2,8 @@ homepage: "https://opensource.zalando.com/skipper/"
 primary_contact: "sandor.szuecs@zalando.de"
 auto_ccs:
   - "oss-fuzz+team-gwproxy@zalando.de"
+  - "sandor.szuecs@zalando.de"
```

Comment:
This one will not have any new impact relative to the `primary_contact`. Is this email a primary email affiliated with a Google account? See https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary

Reply:
yes it's a google company account

Reply:
```
% dig +short MX zalando.de
5 alt1.aspmx.l.google.com.
10 alt4.aspmx.l.google.com.
1 aspmx.l.google.com.
10 alt3.aspmx.l.google.com.
5 alt2.aspmx.l.google.com.
```

Reply:
@DavidKorczynski does a google company account work (we have email, GChat, ..) or do you need a @gmail.com domain?
Normally google IDP just works for all kind of projects with that email address.

Reply:
Google company account works.

Reply:
I get this:
![image](https://github.com/google/oss-fuzz/assets/50872/66efab36-ab37-4880-816f-f70bc2463888)

Reply:
So my question is what do you validate in the OpenID Connect flow?
The email seems to be known because it appears in the error message and the email is the same as configured `primary_contact`.
Do we need to allow something in our google company account to make it work?
The error message shows that it found my email correctly but says "not authorized" so something is wrong either on our side or on yours and I would expect on our side but not sure what this could be.

Reply:
The webinterface access works now. It just took some time.